### DISCUSSION TITLE
Regarding battery level display

### DISCUSSION THREAD

**Discussion by RockyShao** [D0-0K/adwaitapod#36] [Ideas]
I really like this skin, it is now my main skin;
I hope to change the battery icon directly to a percentage; For example, when it comes to displaying the percentage battery level on an iPhone, users have been waiting for a long time.

**Comment by RockyShao** [comment]
In the 'User Preference Settings'

**Comment by D0-0K** [comment]
Hi! Thanks for leavin feedback, I really appreciate it.

So currently, this is not a functional feature within version 3 of adwaitapod. It will be included in version 4, likely to release sometime later next year. So please keep an eye out for it :)

**Reply by RockyShao** [reply]
Got it, ha :)
